pkg/util/kubeobjects/affinity.go

```diff
@@ -14,23 +14,25 @@ const (
 	linux   = "linux"
 )
 
-func AffinityNodeRequirement() []corev1.NodeSelectorRequirement {
-	return affinityNodeRequirementsForArches(amd64)
+func TolerationForAllArches() []corev1.Toleration {
```

Comment:
```suggestion
func TolerationForSupportedArches() []corev1.Toleration {
```
better to name it more explicitly.